.github/workflows/pytest.yml

```diff
@@ -26,10 +26,6 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install
         run: |
-          pip install . pytest pytest-cov
```

Comment:
I don't think these changes should be undone by the present branch. Failed merge?